etc/env/base/chain.toml

```diff
@@ -65,8 +65,11 @@ batch_overhead_l1_gas=800000
 # The maximum amount of gas that can be used by the batch. This value is derived from the circuits limitation per batch.
 max_gas_per_batch=200000000
 
-# The maximum amount of pubdata that can be used by the batch. Note that if the calldata is used as pubdata, this variable should not exceed 128kb.
-max_pubdata_per_batch=100000
+# This will also determine how many blobs are used, when enabled.
+# If the value is less than 126kb -> Calldata or 1 Blob
+# If the value is > 126 kb and < 252kb then 2 blobs are used.
+# Note, the max at this moment is 252kb
+max_pubdata_per_batch=110000
```

Comment:
Let's keep it 100k for now, and change later. We use 100k for all our envs including mainnet for now. Changing this variable affects fees, so let's do it separately